### DISCUSSION TITLE
package dependencies update

### DISCUSSION THREAD

**Discussion by FPEPOSHI** [tectalichq/public-openai-client-php#34] [General]
We are getting following messages on composer. 

Package php-http/message-factory is abandoned, you should avoid using it. Use psr/http-factory instead.
Package spatie/data-transfer-object is abandoned, you should avoid using it. Use spatie/laravel-data instead.

Do you plan to update package dependencies?

**Comment by the-csaba** [comment]
Hi @FPEPOSHI 

Thank you for reaching out and for your concern regarding the package dependencies. We appreciate your vigilance and the time you've taken to inform us about these messages.

We're indeed aware of the deprecation notices for the `php-http/message-factory` and `spatie/data-transfer-object` packages. We're in the process of updating our libraries to use the recommended alternatives.

While we typically don't share our development roadmap publicly, we can assure you that this matter is on our priority list. We're committed to maintaining our software with the most current and secure dependencies.


Csaba

**Comment by FPEPOSHI** [comment]
Hi @om4csaba, do we have any update on this topic?
